test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1130,6 +1144,108 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
             $result.Files[0].ContentType | Should Be 'text/plain'
             $result.Files[0].Content | Should Match 'TestContent'
         }
+
+        It "Verifies Invoke-WebRequest -Form supports string values" {
+            $form = @{TestString = "TestValue"}
+            $uri = Get-WebListenerUrl -Test 'Multipart'
+            $response = Invoke-WebRequest -Uri $uri -Form $form -Method 'POST'
+            $result = $response.Content | ConvertFrom-Json
+
+            $result.Headers.'Content-Type' | Should Match 'multipart/form-data'
+            $result.Items.TestString.Count | Should Be 1
+            $result.Items.TestString[0] | Should Be 'TestValue'
+        }
+
+        It "Verifies Invoke-WebRequest -Form supports a collection of string values" {
+            $form = @{TestStrings = "TestValue", "TestValue2"}
+            $uri = Get-WebListenerUrl -Test 'Multipart'
+            $response = Invoke-WebRequest -Uri $uri -Form $form -Method 'POST'
+            $result = $response.Content | ConvertFrom-Json
+
+            $result.Headers.'Content-Type' | Should Match 'multipart/form-data'
+            $result.Items.TestStrings.Count | Should Be 2
+            $result.Items.TestStrings[0] | Should Be 'TestValue'
```

Comment:
I think `BeExactly` is better for strings in the tests.

Reply:
I will switch these for everything except the file contents which need to remain as match.